{"name": "test_login_0_failed_wrong_datas[data2]", "status": "passed", "attachments": [{"name": "log", "source": "486f00bb-fe1b-4468-96af-b22295a14887-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "b963fb54-ff83-48ee-8c48-dcbec627ecb9-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '186847', 'passwd': 'python', 'check': '请输入正确的手机号'}"}], "start": 1608132623830, "stop": 1608132624102, "uuid": "39b408ac-8ab0-41ca-bd14-020e4001faac", "historyId": "68cf2b0989aa4e51d177a21c2d251b6c", "testCaseId": "735933ecd585cb26d239d15346a80fa1", "fullName": "testcase.test_1login_pytest.TestLogin#test_login_0_failed_wrong_datas", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_1login_pytest"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-J6CF24D"}, {"name": "thread", "value": "53180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_1login_pytest"}]}